mumbleBot.py

```diff
@@ -252,20 +252,16 @@ def message_received(self, text):
             # you want to split the username
             user = user.split()[0]
 
-        if message.startswith(tuple(var.config.get('commands', 'command_symbol'))):
-            # remove the symbol from the message
-            message = message[1:].split(' ', 1)
-
-            # use the first word as a command, the others one as  parameters
-            if len(message) > 0:
-                command = message[0].lower()
-                parameter = ''
-                if len(message) > 1:
-                    parameter = message[1].rstrip()
-            else:
+        command_symbols = var.config.get('commands', 'command_symbol')
+        match = re.match(fr'^[{re.escape(command_symbols)}](?P<command>\S+)(?:\s(?P<argument>.*))?', message)
+        if match:
+            command = match.group("command").lower()
+            argument = match.group("argument")
```

Comment:
```suggestion
            argument = match.group("argument") or ""
```

Which would make it default to empty string as expected by other parts of the codebase. See #278.